etc/profile-m-z/transmission-qt.profile

```diff
@@ -11,8 +11,8 @@ private-bin transmission-qt
 
 # private-lib - breaks on Arch
 ignore private-lib
-
 ignore memory-deny-write-execute
+protocol netlink
```

Comment:
```suggestion

protocol netlink
```

What does the absence of `netlink` break?

Reply:
Could not create AF_NETLINK socket (Operation not supported)

Reply:
But nothing is broken, right? That's Qt. Why should we add unnecessary attack-surface?

Reply:
I don't know whether Qt prints that. Nothing seems broken yet. But, I didn't like error messages.

Reply:
@crocket [on Feb 15](https://github.com/netblue30/firejail/pull/4948#discussion_r806919223):

> I don't know whether Qt prints that. Nothing seems broken yet. But, I didn't
> like error messages.
> 
> Why does Qt try to create a netlink socket?

Related thread:

* <https://github.com/netblue30/firejail/issues/4925#issuecomment-1039199543>

Reply:
I deleted modification to transmission-qt.profile.